app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,44 @@
+class GuestsController < ApplicationController
+  before_action :set_guest, only: [:destroy]
+  before_action :set_event
+
+
```

Comment:
I'm not sure why it's indented weirdly, it's a little confusing, maybe just fix this